src/components/Card/StandardCard/styles.scss

```diff
@@ -11,6 +12,10 @@
   width: 15rem;
 }
 
+p.std-card-title {
+  @include type.type-style('heading-03');
+}
+
 .std-card-icon {
```

Comment:
I think this rule can be removed, there is no class `std-card-icon` on the StandardCard component.